internal/output/markdown.go

```diff
@@ -207,6 +211,13 @@ func ToMarkdown(out Root, opts Options, markdownOpts MarkdownOptions) ([]byte, e
 		}
 	}
 
+	erroredProjectCount := 0
```

Comment:
Should we exclude anything with `len(p.Metadata.Errors) == 0` from the skipped project count so the comment just shows:

`1 project could not be evaluated`.

Reply:
👍